apps/ledger-live-desktop/src/renderer/families/cosmos/TransactionConfirmFields.jsx

```diff
@@ -73,20 +71,23 @@ const onExternalLink = (account, address) => {
   if (URL) openURL(URL);
 };
 
-const CosmosDelegateValidatorsField = ({
+export const CosmosDelegateValidatorsField = ({
   account,
   parentAccount,
   transaction,
   field,
 }: FieldComponentProps) => {
   const mainAccount = getMainAccount(account, parentAccount);
-
-  invariant(transaction.family === "cosmos", "cosmos transaction");
+  invariant(
+    transaction.family === "cosmos" || transaction.family === "osmosis",
```

Comment:
Yeah I agree I would like to brainstorm how to better refactor the staking UIs in LLD. It would be cool to come up with a more consistent strategy for implementing all Cosmos-based cryptos. For the time being I tried to find a balance between duplicating code and re-using and extending existing Cosmos code. Onward!